### PR TITLE
Surgery kits

### PR DESCRIPTION
**Pull request by calibraptor**
yeheheh
- Adds purple first aid boxes that contain surgery tools.
- A bonesaw has been implemented for the purposes of having a tool with the utility of the surgical saw, but none of the OP damage. Plus it fits inside boxes.
- Two surgery kits have been placed in medical storage, and one in robotics.

### CODE REVIEW

**Comment by GenericGoose** [comment]
good idea, though it doesn't make sense to me having tools in a box like that

**Comment by ArcLumin** [comment]
I think it's a good idea, shit gets stolen far too often and makes it so you can only ghetto surgery or try to hack the robotics vendor

**Comment by Kierany9** [comment]
That hideously oversaturated purple though...

File: code/modules/surgery/tools.dm [hunk 1]
```diff
@@ -84,7 +84,7 @@
 	return (BRUTELOSS)
 
 
-/obj/item/weapon/circular_saw
+/obj/item/weapon/circular_saw/
```
Comment (JohnGinnane) [inline code/modules/surgery/tools.dm]: Isn't this a big no no, having / at the end of object path?

**Comment by GenericGoose** [comment]
pimp
put rhinestones on the saw

**Comment by ghost** [comment]
People stealing circular saws for robusting is pretty cancerous.
I like it +1.

File: code/modules/surgery/tools.dm (hunk 1 above)
Comment (Gandalf2k15) [inline code/modules/surgery/tools.dm]: aaaaaaaaa what u doin why the / fixxxxxx

**Comment by Spacedong** [comment]
I don't really like it, it simplifies surgery and shitlords will just steal the kit and put it in their pack making medical doctors useless. If you don't want people taking the saws, put them in your backpack.

**Comment by dreamcrab** [comment]
as long as it doesnt get rid of the original circular saw and all the laid out tools i am alllllll for this

fuck yea +66666

**Comment by Kierany9** [comment]
Why don't you put a surgery tool vendor in the surgery room instead? It'd accomplish the same thing and be less acccessible to the greyshirts who inevitably break into medbay anyway.

**Comment by dreamcrab** [comment]
not the worst idea tbh

fun new medivend in medical storage?

**Comment by calibraptor** [comment]
I'd have to sprite a new vending machine for it, but tbh that'd be a neat idea too. That said, this has the requisite approvals for mergan

**Comment by Carbonhell** [comment]
I don't really like the idea a lot, but heads decided so. Code's ok, merging(Even through the sprite facing left triggers me a bit,but i saw the circular saw does too so...)

**Comment by Carbonhell** [comment]
NEVERMIND lacks a changelog

**Comment by calibraptor** [comment]
I don't see why having spare surgery tools in the medbay is a bad thing. That said, added a changelog.

**Pull request merged**